apps/files/src/components/AllFilesList.vue

```diff
@@ -29,14 +29,7 @@
                 class="uk-margin-small-left"
               />
             </div>
-            <div>
-              <oc-button v-if="$_isUserShare(item)" class="file-row-share-indicator uk-text-middle" :aria-label="$_shareUserIconLabel(item)" @click="$_openSideBar(item, 'files-sharing')" variation="raw">
-                <oc-icon name="group" class="uk-text-middle" size="small" :variation="$_shareUserIconVariation(item)"/>
-              </oc-button>
-              <oc-button v-if="$_isLinkShare(item)" class="file-row-share-indicator uk-text-middle" :aria-label="$_shareLinkIconLabel(item)" @click="$_openSideBar(item, 'file-link')" variation="raw">
-                <oc-icon name="link" class="uk-text-middle" size="small" :variation="$_shareLinkIconVariation(item)"/>
-              </oc-button>
-            </div>
+            <StatusIndicators :item="item" :currentFolderPath="currentFolder.path" @click="$_openSideBar" />
```

Comment:
Shouldn't it rather be `parentPath` since it's the parent of the item ?
For context, think about flat lists where each item might have a different parent folder (like favorites), so the name "current folder" is misleading

Should the property be called `resource` as this is either a file or folder ?

Reply:
Right. Took it from the state originally and haven't realised the other use cases. Will change it. For the resource though I usually use `item` which seems to me as a good enough alternative. I can change that though to resource. Don't really care which one it will be.